package.json

```diff
@@ -116,7 +117,11 @@
       "src/**/**.{js,jsx}",
       "!src/**/*.stories.jsx",
       "!src/setupTest.js"
-    ]
+    ],
+    "moduleNameMapper": {
+      ".+\\.(css|styl|less|sass|scss)$": "identity-obj-proxy",
+      ".+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/fileTransformer.js"
```

Comment:
@Syed-Ali-Abbas-Zaidi, do you mind helping me understand why this is necessary after the upgrade?

Reply:
@arbrandes, We upgraded paragon from `v12` -> `v20` and in this major version upgrade there were some `non-js` files were added to paragon (like `svg` and other stuff). Due to this, the tests were failing as jest could not parse non-js files. That is why we needed to transform these files so that jest was able to parse them.

Reply:
Ok.  This might be an artifact of not using Enzyme in this component, I guess.  Still a bit surprising, though, as I don't see this transformer necessary anywhere else.

Before approving, @abdullahwaheed, care to give your take on this one?

Reply:
Hi @arbrandes, this MFE isn't using our base `frontend-build` which resides such [configurations](https://github.com/openedx/frontend-build/blob/master/config/jest.config.js#L20). That's why we used this approach.  Although we can make this MFE to use `frontend-build` but that might result in huge changes.

@Syed-Ali-Abbas-Zaidi could you please move these configurations to `jest.config.js` as we have in [frontend-app-learner-portal-programs](https://github.com/openedx/frontend-app-learner-portal-programs/blob/master/jest.config.js)?

Reply:
@abdullahwaheed, As per your suggestion I have created a separate `config` file for jest configurations.